bridge/client/app/_views/ktb-settings-view/ktb-project-settings/ktb-project-settings.component.spec.ts

```diff
@@ -216,7 +345,8 @@ describe('KtbProjectSettingsComponent', () => {
     fixture.detectChanges();
 
     // then
-    expect(component.canDeactivate()).not.toEqual(true);
+    expect(component.canDeactivate() instanceof Observable).toEqual(true);
+    expect(spy).not.toHaveBeenCalled();
```

Comment:
The spy will never be called if a new observable is returned. Anyway, this is a bit of weird behavior. I would expect that a function call always returns a new observable and then if I don't subscribe to it inside this test, it will not be subscribed as this is a completely new result/object of `canDeactivate()`

Reply:
Agree, as discussed I will remove this test.